tmail-backend/jmap/extensions/src/main/java/com/linagora/tmail/james/jmap/contact/OpenPaasContactsConsumer.java

```diff
@@ -92,29 +96,43 @@ public Flux<AcknowledgableDelivery> delivery() {
 
     private void messageConsume(AcknowledgableDelivery ackDelivery, String messagePayload) {
         Mono.just(messagePayload)
-            .map(message -> gson.fromJson(message, OpenPaasContactMessage.class))
+            .<ContactAddedRabbitMqMessage>handle((message, sink) -> {
```

Comment:
We likely do not need that "just" and can inline the reactor transformations.

Reply:
Not sure how...